ndcube/tests/test_ndcube.py

```diff
@@ -258,58 +258,58 @@
        'hello': {'axis': 1, 'value': u.Quantity(range(int(cube.dimensions[1].value)), unit=u.pix)},
        'bye': {'axis': 2, 'value': u.Quantity(range(int(cube.dimensions[2].value)), unit=u.pix)}}
       ),
-     (cube[1, 1],
+     (cube[0, 1],
```

Comment:
What was the reason for changing this?  Was it just because the ```1``` wasn't working?  If so we should understand why it isn't.

Reply:
Yeah, `1` was no longer working

Reply:
Why?